### PR TITLE
Force field section draft

### PR DESCRIPTION
**Pull request by JIMonroe**
I've created a draft of the force fields section. Hopefully this ties in well with intermolecular potentials.

### CODE REVIEW

**Comment by JIMonroe** [comment]
Also, sorry if I stepped on anyones toes here.  I was just looking to contribute a section I felt comfortable writing up in a short period of time and went for this one.  We still need to write sections on barostats and MD integrators, so there's lots of material left!

**Comment by avisekdas** [comment]
Hi Jacob,

Thanks for taking the initiative. Please go ahead with your proposed plans,
if nobody objects. I am still tied up with a lot of things. I will get back
to it as soon as I can find some time. I got an email from Michael Shirt
and it looked like other groups were also behind schedule.

Thanks again.

Avisek

**Comment by davidlmobley** [comment]
Excellent, Jacob, please proceed. I can spend some time working on other aspects in the near future; I was mainly waiting to see that I wasn't going to be the only one carrying the load if I tried to work towards finishing this. 

I'll go ahead and merge this in and we can edit in a separate PR. You and I should probably do as much as we can on our own and then get others to review/pitch in as they can. We've got one month!

File: paper/basic_training.tex [hunk 1]
```diff
@@ -488,13 +495,76 @@ \subsubsection{Online resources}
...
+%JIM: Not sure if it's important, but I wanted to include something
+%about force field terms like CMAP, as well as external fields. The 
+%below sentence is an attempt at that, but it may not be needed.
+This may even include empirical corrections based on discrete binning 
+along a particular set of degrees of freedom, as well as applied external  
+fields (i.e. electric fields) and force field terms describing the effect of 
+degrees of freedom, such as solvent, that have been removed from the system via 
+"coarse-graining." For a more in-depth discussion of common (and even less common) force field terms, see Ch. 4 of \citet{LeachBook}.
+
+Functional forms used to describe specific terms in a potential energy function
```
Comment (davidlmobley) [inline paper/basic_training.tex]: Not sure I follow what you're getting at here in this sentence beginning "Functional forms..."; is this comparing different force field families? Maybe an example would help?

**Pull request merged**